suite-native/module-settings/src/components/FAQInfoPanel.tsx

```diff
@@ -17,6 +20,250 @@ const AccordionContentText = ({
     </Text>
 );
 
+const BluetoothAndroidFAQ = ({ coinLabel }: { coinLabel: string }) => (
```

Comment:
This might be easier to read.
```suggestion
const BluetoothAndroidFaq = ({ coinLabel }: { coinLabel: string }) => (
```

Reply:
I don't want to change it now because I find FAQ more readable than Faq. But it would be nice to have some convention decided, so I created a code improvement proposal https://github.com/trezor/trezor-suite/issues/22239